src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAO.scala

```diff
@@ -119,11 +119,11 @@ class ElasticSearchDAO(client: TransportClient, indexName: String, ontologyDAO:
   }
 
   override def findDocuments(criteria: LibrarySearchParams, groups: Seq[String]): Future[LibrarySearchResponse] = {
-    findDocumentsWithAggregateInfo(client, indexName, criteria, groups, ontologyDAO)
+    findDocumentsWithAggregateInfo(client, indexName, criteria, groups, researchPurposeDAO)
```

Comment:
More plumbing. You'll see a lot of swapping out `ontologyDAO` for `researchPurposeDAO` in this PR. I wish we had an easier way to inject dependencies instead of manually passing them around everywhere.

Reply:
Could we could pass them as implicits?

Reply:
👎 to using implicits unless necessary. Agreed that DI is a little more painful here.